<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Selector</class>
 <widget class="QWidget" name="Selector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Selector</string>
  </property>
  <widget class="QComboBox" name="condBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>恒定转速</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>恒定功率</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>10</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>条件</string>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>201</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="1">
     <widget class="QLineEdit" name="condInput">
      <property name="text">
       <string>3000</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="condLabel">
      <property name="text">
       <string>转速/RPM</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>空速/(m/s)</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="speedInput">
      <property name="text">
       <string>7.0</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>静拉力&gt;(牛)</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="staticThrustInput">
      <property name="text">
       <string>10</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>动拉力&gt;(牛)</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="dynamicThrustInput">
      <property name="text">
       <string>7</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="wattLabel">
      <property name="text">
       <string>功率&lt;(瓦)</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="powerInput">
      <property name="text">
       <string>180</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="etaLabel">
      <property name="text">
       <string>空速下效率&gt;(%)</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="etaInput">
      <property name="text">
       <string>50</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="selectButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>270</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>筛选</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>60</y>
     <width>171</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AcadEref</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>计算过程可能会消耗20秒，请耐心等待，
不要重复点击“筛选”按钮！！！</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>condBox</tabstop>
  <tabstop>condInput</tabstop>
  <tabstop>speedInput</tabstop>
  <tabstop>staticThrustInput</tabstop>
  <tabstop>dynamicThrustInput</tabstop>
  <tabstop>etaInput</tabstop>
  <tabstop>powerInput</tabstop>
  <tabstop>selectButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
